packages/cdk/lib/env/context-app-parameters.ts

```diff
@@ -87,6 +87,14 @@ export class ContextAppParameters {
    * The types of EC2 instances that may be launched in the compute environment.
    */
   public readonly instanceTypes?: InstanceType[];
+  /**
+   * The maximum number of Amazon EC2 vCPUs that an environment can reach for the nextflow engine.
+   */
+  public readonly vCpus?: number;
+  /**
+   * The maximum memory limit that an environment can reach for the nextflow engine.
```

Comment:
Is this NextFlow specific? How it will affect other supported engines? Have we tested this change with other engines?

Reply:
Yes this is NextFlow specific.

Reply:
Should you update comments than?